inginious/frontend/templates/mycourses.html

```diff
@@ -49,70 +49,266 @@ <h3>{{ _("Last tried exercises") }}</h3>
+                                        {% if course.is_lti() %}
+                                            {{ _("LTI course") }} -
+                                        {% elif course.is_archive() %}
+                                            {{ _("Archive") }} - {{show_time(course.get_archiving_date())}}
+                                        {% elif not course.is_open_to_non_staff() %}
+                                            {{ _("Hidden course") }} -
+                                        {% endif %}
+
+                                        {% if username in course.get_admins() or user_manager.user_is_superadmin() %}
+                                            {{ _("Administrator") }}
+                                        {% elif username in course.get_tutors() %}
+                                            {{ _("Tutor") }}
+                                        {% elif user_manager.course_is_user_registered(course) %}
+                                            {{ _("Student") }}
+                                        {% endif %}
+                                    </i>
+                                </div>
+                            </div>
+                        </a>
+                    {% endfor %}
+                {% else %}
+                    <a href="#register" >{{ _("You are not registered to any course") }}</a>
+                {% endif %}
+            </div>
+            {% if user_manager.user_is_superadmin() %}
+                <hr/>
+                {% if success == True %}
+                    <div class="alert alert-success alert-dismissable" role="alert">
+                        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+                        {{ _("Course created.") }}
                     </div>
-                </div>
-            </a>
+                {% elif success == False %}
+                    <div class="alert alert-danger alert-dismissable" role="alert">
+                        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+                        {{ _("Failed to create the course.") }}
+                    </div>
+                {% endif %}
+                <form class="form row" role="form" method="post">
+                    <div class="form-group col-sm-8">
+                        <label class="sr-only" for="new_courseid">{{ _("Course") }}</label>
+                        <input name="new_courseid" type="text" class="form-control" id="new_courseid" placeholder="{{ _('New course id') }}">
+                    </div>
+                    <div class="form-group col-sm-4">
+                        <button type="submit" class="btn btn-block btn-primary">{{ _("Create new course") }}</button>
+                    </div>
+                </form>
+            {% endif %}
+
+
+<script>
+    $(document).on('click', '.course_pin', function(event) {
```

Comment:
Put as maximum script as possible in the standalone JS files so that they can be minified. If data from the page has to be used in the code, use the `data-*`attribute of the element triggering the event.